kotsadm/web/src/components/AirgapUploadProgress.jsx

```diff
@@ -38,13 +38,24 @@ class AirgapUploadProgress extends React.Component {
       });
 
       const response = await res.json();
+
+      if (!response.ok) {
+        this.setState({
+          installStatus: "airgap_upload_error",
+          currentMessage: `Encountered an error while uploading: ${response.error}`
+        });
+      } else {
+        this.setState({
+          installStatus: response.installStatus,
+          currentMessage: response.currentMessage,
+        });
+      }
       
-      this.setState({
-        installStatus: response.installStatus,
-        currentMessage: response.currentMessage,
-      });
     } catch(err) {
-      console.log(err);
```

Comment:
should we also keep this `console.log` as well? the error object has more details than just the `message`, which can be helpful sometimes.